crates/bin/pindexer/src/dex_ex/schema.sql

```diff
@@ -1,19 +1,3 @@
-CREATE TABLE IF NOT EXISTS dex_ex_candlesticks (
```

Comment:
Breaking schema change for the [dex-explorer](https://github.com/penumbra-zone/dex-explorer), but worth it, because these changes are intended to ease dex explorer development. I'll file an issue over in that repo to track the required change.

Reply:
> I'll file an issue over in that repo to track the required change.

https://github.com/penumbra-zone/dex-explorer/issues/132

Reply:
As a further bit of context, the reason for the breaking change there is that it requires several more writes to move the candlesticks into a separate table, versus just writing them inline ; given the pre-alpha nature of the explorer a breaking change for simpler and more performant code forever is a good tradeoff